### ISSUE TITLE
Preview TiddlyText

### ISSUE THREAD

**Issue by mardukbp** [PaulPorfiroff/atom-language-tiddlywiki5#1]
It would be great to be able to preview the TiddlyWiki markup like it is currently possible for Markdown.

**Comment by PaulPorfiroff** [comment]
@mardukbp, Yeah, I thought about this nice feature, too, so thanks for refresher.

I'm still not sure it really worth efforts.

Here are my cons:
- To handle TW5 widgets (e.g. transclusion, image, list, etc.) you will often need the _entire_ wiki store at hand, so will have to mess up with detecting wiki files (think of e.g. 'Untitled' tab, where you don't know which wiki tiddler belongs to), loading different wiki formats, managing store.
- The resulting markup will be static, as unsafe attributes and tags will be stripped off.

On the other hand, you can easily setup live updated interactive preview using web GUI or move it into Atom's pane (e.g. [browser-plus](https://atom.io/packages/browser-plus)).

Share your thoughts.

**Comment by mardukbp** [comment]
I was mostly thinking of the intersection between TiddlyText and
Markdown. But given that Atom is still too slow compared with Emacs
(specially the startup time), I already implemented what I needed in the
latter and no longer need the TW web UI.
